src/losses/perceptual.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, resize=False):
         self.register_buffer("mean", torch.tensor([0.485, 0.456, 0.406]).view(1, 3, 1, 1))
         self.register_buffer("std", torch.tensor([0.229, 0.224, 0.225]).view(1, 3, 1, 1))
 
-    def forward(self, input, target, feature_layers=[0, 1, 2, 3], style_layers=[]):
+    def forward(self, input, target, feature_layers=[0, 1, 2, 3], style_layers=[1,2]):
```

Comment:
It's not the best idea to use `style_layers` here because gram loss is not normalised here. Check Gram loss for reference: https://github.com/grenlayk/text-deep-fake/blob/53f9d2feb0752845c8a99ae0c1f99e679e207943/src/losses/gram.py#L44